MFVNeutralino/python/Vertexer_cfi.py

```diff
@@ -26,7 +26,7 @@
                                track_genjet_match_thres = cms.double(0.01),
                                min_genparticle_pt = cms.double(0),
                                verbose = cms.untracked.bool(False),
-                               histos = cms.untracked.bool(True),
+                               histos = cms.untracked.bool(False),
```

Comment:
Do the histos take up too much space now? We rely on some of them during template construction, and can be useful to have.